# pre-defined poses for calibration purposes
# angles in degrees

legs straight / elbows bent:
  r_arm_sh_p1 :   0.0           # r_arm_sh_p1
  l_arm_sh_p1 :   0.0           # l_arm_sh_p1
  r_arm_sh_r :   0.0           # r_arm_sh_r
  l_arm_sh_r :   0.0           # l_arm_sh_r
  r_arm_sh_p2 :   0.0           # r_arm_sh_p2
  l_arm_sh_p2 :   0.0           # l_arm_sh_p2
  r_arm_el_y :   0.0           # r_arm_el_y
  l_arm_el_y :   0.0           # l_arm_el_y
  r_arm_wr_r :   0.0           # r_arm_wr_r
  l_arm_wr_r :   0.0           # l_arm_wr_r
  r_arm_wr_y :   0.0           # r_arm_wr_y
  l_arm_wr_y :   0.0           # l_arm_wr_y
  r_arm_wr_p :   0.0           # r_arm_wr_p
  l_arm_wr_p :   0.0           # l_arm_wr_p
  r_leg_hip_y :   0.0           # r_leg_hip_y
  l_leg_hip_y :   0.0           # l_leg_hip_y
  r_leg_hip_r :   0.0           # r_leg_hip_r
  l_leg_hip_r :   0.0           # l_leg_hip_r
  r_leg_hip_p :  -6.0           # r_leg_hip_p 
  l_leg_hip_p :   6.0           # l_leg_hip_p
  r_leg_kn_p :    9.0           # r_leg_kn_p
  l_leg_kn_p :   -9.0           # l_leg_kn_p
  r_leg_an_p :    6.0           # r_leg_an_p
  l_leg_an_p :   -6.0           # l_leg_an_p
  r_leg_an_r :   0.0           # r_leg_an_r
  l_leg_an_r :   0.0           # l_leg_an_r
  torso_y :   0.0           # torso_y
  head_y :   0.0           # head_y
  head_p :   0.0           # head_p

legs straight / elbows straight:
  r_arm_sh_p1 :   0.0           # r_arm_sh_p1
  l_arm_sh_p1 :   0.0           # l_arm_sh_p1
  r_arm_sh_r :   0.0           # r_arm_sh_r
  l_arm_sh_r :   0.0           # l_arm_sh_r
  r_arm_sh_p2 :   0.0           # r_arm_sh_p2
  l_arm_sh_p2 :   0.0           # l_arm_sh_p2
  r_arm_el_y :   90.0           # r_arm_el_y
  l_arm_el_y :   -90.0           # l_arm_el_y
  r_arm_wr_r :   0.0           # r_arm_wr_r
  l_arm_wr_r :   0.0           # l_arm_wr_r
  r_arm_wr_y :   0.0           # r_arm_wr_y
  l_arm_wr_y :   0.0           # l_arm_wr_y
  r_arm_wr_p :   0.0           # r_arm_wr_p
  l_arm_wr_p :   0.0           # l_arm_wr_p
  r_leg_hip_y :   0.0           # r_leg_hip_y
  l_leg_hip_y :   0.0           # l_leg_hip_y
  r_leg_hip_r :   0.0           # r_leg_hip_r
  l_leg_hip_r :   0.0           # l_leg_hip_r
  r_leg_hip_p :  -6.0           # r_leg_hip_p 
  l_leg_hip_p :   6.0           # l_leg_hip_p
  r_leg_kn_p :    9.0           # r_leg_kn_p
  l_leg_kn_p :   -9.0           # l_leg_kn_p
  r_leg_an_p :    6.0           # r_leg_an_p
  l_leg_an_p :   -6.0           # l_leg_an_p
  r_leg_an_r :   0.0           # r_leg_an_r
  l_leg_an_r :   0.0           # l_leg_an_r
  torso_y :   0.0           # torso_y
  head_y :   0.0           # head_y
  head_p :   0.0           # head_p

legs bent / elbows bent:
  r_arm_sh_p1 :   0.0           # r_arm_sh_p1
  l_arm_sh_p1 :   0.0           # l_arm_sh_p1
  r_arm_sh_r :   0.0           # r_arm_sh_r
  l_arm_sh_r :   0.0           # l_arm_sh_r
  r_arm_sh_p2 :   0.0           # r_arm_sh_p2
  l_arm_sh_p2 :   0.0           # l_arm_sh_p2
  r_arm_el_y :   0.0           # r_arm_el_y
  l_arm_el_y :   0.0           # l_arm_el_y
  r_arm_wr_r :   0.0           # r_arm_wr_r
  l_arm_wr_r :   0.0           # l_arm_wr_r
  r_arm_wr_y :   0.0           # r_arm_wr_y
  l_arm_wr_y :   0.0           # l_arm_wr_y
  r_arm_wr_p :   0.0           # r_arm_wr_p
  l_arm_wr_p :   0.0           # l_arm_wr_p
  r_leg_hip_y :   0.0           # r_leg_hip_y
  l_leg_hip_y :   0.0           # l_leg_hip_y
  r_leg_hip_r :   0.0           # r_leg_hip_r
  l_leg_hip_r :   0.0           # l_leg_hip_r
  r_leg_hip_p :   0.0           # r_leg_hip_p 
  l_leg_hip_p :   0.0           # l_leg_hip_p
  r_leg_kn_p :   0.0           # r_leg_kn_p
  l_leg_kn_p :   0.0           # l_leg_kn_p
  r_leg_an_p :   0.0           # r_leg_an_p
  l_leg_an_p :   0.0           # l_leg_an_p
  r_leg_an_r :   0.0           # r_leg_an_r
  l_leg_an_r :   0.0           # l_leg_an_r
  torso_y :   0.0           # torso_y
  head_y :   0.0           # head_y
  head_p :   0.0           # head_p 

legs bent / elbows straight:
  r_arm_sh_p1 :   0.0           # r_arm_sh_p1
  l_arm_sh_p1 :   0.0           # l_arm_sh_p1
  r_arm_sh_r :   0.0           # r_arm_sh_r
  l_arm_sh_r :   0.0           # l_arm_sh_r
  r_arm_sh_p2 :   0.0           # r_arm_sh_p2
  l_arm_sh_p2 :   0.0           # l_arm_sh_p2
  r_arm_el_y :   90.0           # r_arm_el_y
  l_arm_el_y :   -90.0           # l_arm_el_y
  r_arm_wr_r :   0.0           # r_arm_wr_r
  l_arm_wr_r :   0.0           # l_arm_wr_r
  r_arm_wr_y :   0.0           # r_arm_wr_y
  l_arm_wr_y :   0.0           # l_arm_wr_y
  r_arm_wr_p :   0.0           # r_arm_wr_p
  l_arm_wr_p :   0.0           # l_arm_wr_p
  r_leg_hip_y :   0.0           # r_leg_hip_y
  l_leg_hip_y :   0.0           # l_leg_hip_y
  r_leg_hip_r :   0.0           # r_leg_hip_r
  l_leg_hip_r :   0.0           # l_leg_hip_r
  r_leg_hip_p :   0.0           # r_leg_hip_p 
  l_leg_hip_p :   0.0           # l_leg_hip_p
  r_leg_kn_p :   0.0           # r_leg_kn_p
  l_leg_kn_p :   0.0           # l_leg_kn_p
  r_leg_an_p :   0.0           # r_leg_an_p
  l_leg_an_p :   0.0           # l_leg_an_p
  r_leg_an_r :   0.0           # r_leg_an_r
  l_leg_an_r :   0.0           # l_leg_an_r
  torso_y :   0.0           # torso_y
  head_y :   0.0           # head_y
  head_p :   0.0           # head_p 

